project2/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Project 2: C O M P S </title>
+    <link href="https://fonts.googleapis.com/css?family=Be+Vietnam:300,600,800&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+    <section>
+        <img src="images/featured-film-example.jpg" class="articlepicture"
+            alt="Everything is Illuminated movie screenshot, guy staring at a wall with hanging objects">
```

Comment:
This is good, but I offer a bit of refinement:
```suggestion
            alt="A still from the movie Everything is Illuminated where a guy in a suit stares at a wall covered with hanging objects.">
```